### PR TITLE
shorten genesis

### PR DESCRIPTION
**Pull request by noamnelke**

### CODE REVIEW

File: oracle/blockoracle.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func (bo *MinerBlockOracle) calcEligibilityProofs(epochNumber types.EpochId) err
 		activeSetSize = bo.committeeSize
 	} else {
 		activeSetSize = atx.ActiveSetSize
-		if epochNumber.IsGenesis() && activeSetSize < bo.committeeSize {
+		if atx.PubLayerIdx.GetEpoch(bo.layersPerEpoch).IsGenesis() && activeSetSize < bo.committeeSize {
```
Comment (antonlerner) [inline oracle/blockoracle.go:83]: same outcome since atx is fetched by publication epoch

Comment (noamnelke) [reply oracle/blockoracle.go:83]: nope, it fetches by `TargetEpoch`, not `PublicationEpoch`.

File: oracle/blockeligibilityvalidator.go [hunk 2]
```diff
@@ -42,6 +42,9 @@ func (v BlockEligibilityValidator) BlockEligible(block *types.BlockHeader) (bool
 			return false, err
 		}
 		activeSetSize = atx.ActiveSetSize
+		if atx.PubLayerIdx.GetEpoch(v.layersPerEpoch).IsGenesis() && activeSetSize < v.committeeSize {
```
Comment (antonlerner) [inline oracle/blockeligibilityvalidator.go:45]: this should not happen

Comment (noamnelke) [reply oracle/blockeligibilityvalidator.go:45]: of course it should. We publish `activeSetSize = 0` in genesis. We can actually skip the second part of the condition (it was added to allow ATXs in previous epoch even in genesis, but not relevant with 1 epoch genesis).

Comment (noamnelke) [reply oracle/blockeligibilityvalidator.go:45]: btw, we could change this and publish the hard-coded active set size.

**Comment by antonlerner** [comment]
lets set this aside for now, at least until we have reviewed the genesis flow again

**Pull request closed without merging**